README.md

```diff
@@ -1,18 +1,22 @@
 
 # Stackdriver Sandbox (Alpha)
 
-Stackdriver Sandbox is an open-source tool that helps practitioners to learn Service Reliability Engineering practices from Google and apply them on their cloud services using Stackdriver. It is based on [Hipster Shop](https://github.com/GoogleCloudPlatform/microservices-demo), a cloud-native microservices application.
+Stackdriver Sandbox is an open-source tool that helps practitioners to learn Service Reliability Engineering practices from Google and apply them on their cloud services using [Operations](https://cloud.google.com/products/operations) (formerly Stackdriver).
+It is based on [Hipster Shop](https://github.com/GoogleCloudPlatform/microservices-demo), a cloud-native microservices application.
 
 Sandbox offers:
 
 * **Demo Service** - an application built using microservices architecture on modern, cloud-native stack.
 * **One-click deployment**  - a script handles the work of deploying the service to Google Cloud Platform.
 * **Load Generator** - a component that produces synthetic traffic on a demo service.
-* (Soon) **SRE Runbook** - pre-built routine procedures for operating the deployed sample service that follow best SRE practices using Stackdriver.
+* (Soon) **SRE Runbook** - pre-built routine procedures for operating the deployed sample service that follow best SRE practices using Operations.
```

Comment:
This is tricky, but when we refer to the suite of tools that was formerly known as Stackdriver, we should call it "Ops Management" or "Operations Management".

Reply:
done